jaxley/modules/cell.py

```diff
@@ -203,17 +210,26 @@ def _init_morph_jax_spsolve(self):
         """
 
         # Edges between compartments within the branches.
-        # `[offset, offset, 0]` because we want to offset `source` and `sink`, but
-        # not `type`.
         self._comp_edges = pd.concat(
             [
-                [offset, offset, 0] + branch._comp_edges
-                for offset, branch in zip(self.cumsum_nseg, self.branch_list)
+                pd.DataFrame().from_dict(
+                    {
+                        "source": list(range(cumsum_nseg, nseg - 1 + cumsum_nseg))
+                        + list(range(1 + cumsum_nseg, nseg + cumsum_nseg)),
+                        "sink": list(range(1 + cumsum_nseg, nseg + cumsum_nseg))
+                        + list(range(cumsum_nseg, nseg - 1 + cumsum_nseg)),
```

Comment:
do we want lists specifically or isnt it better to replace `list(range(...))` by `np.arange`

Reply:
using + to concat the lists is handy here :)